repairnator/repairnator-pipeline/src/test/java/fr/inria/spirals/repairnator/process/step/TestAbstractStep.java

```diff
@@ -103,14 +103,14 @@ public void testGetPomOnProjectWithSubModule() throws IOException, InterruptedEx
 
         ProjectInspector mockInspector = mock(ProjectInspector.class);
 
-        String localRepoPath = tmpDir.getAbsolutePath() + "/repairnator";
+        String localRepoPath = tmpDir.getAbsolutePath() + "/repairnator/repairnator";
```

Comment:
Shouldn't it be `repairnator/maven-repair` now, I got that from the Travis failure:

```
  TestAbstractStep.testGetPomOnProjectWithSubModule:116 
Expected: is "/tmp/test_get_pom_project_with_submodule4188477295756336730/repairnator/repairnator/pom.xml"
     but: was "/tmp/test_get_pom_project_with_submodule4188477295756336730/repairnator/maven-repair/pom.xml"
```

Reply:
that would have worked too